README.md

```diff
@@ -150,15 +172,25 @@ data: 'username=fzipi
 
 Other interesting functions you can use are: `randBytes`, `htpasswd`, `encryptAES`, etc.
 
-## Overriding test results
+## Overriding tests
 
 Sometimes you have tests that work well for some platform combinations, e.g. Apache + modsecurity2, but fail for others, e.g. NGiNX + modsecurity3. Taking that into account, you can override test results using the `testoverride` config param. The test will be run, but the _result_ would be overriden, and your comment will be printed out.
 
-Example:
+Tests can be altered using four lists:
+- `input` allows you to override global parameters in tests. An example usage is if you want to change the `dest_addr` of all tests to point to an external IP or host.
+- `ignore` is for tests you want to ignore. It will still execute the test, but ignore the results. You should add a comment on why you ignore the test
```

Comment:
```suggestion
- `ignore` is for tests you want to ignore. It will still execute the test, but ignore the result. You should add a comment on why you ignore the test
```